src/styles/modules/card.scss

```diff
@@ -0,0 +1,135 @@
+$text-color: #616070;
+$title-color: #060b35;
+$color-button: #00acdc;
+
+.body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+p {
+  margin: 0;
+  font-weight: 700;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition:
+    transform 300ms ease,
+    color 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    box-sizing: border-box;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $title-color;
+    margin-bottom: 4px;
+    transition: color 300ms ease;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__subtitle {
+    box-sizing: border-box;
+    color: #616070;
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__response {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 166px;
+    margin: 0 0 24px;
+
+    .stars {
+      display: flex;
+
+      &__star {
+        background-image: url('../images/star.svg');
+        background-repeat: no-repeat;
+        background-position: center;
+        margin-right: 4px;
+        width: 16px;
+        height: 16px;
+
+        &:nth-child(-n + 4) {
+          background-image: url('../images/star-active.svg');
```

Comment:
Ensure that the images 'star.svg' and 'star-active.svg' exist in the '../images/' directory. If these images are missing, the star ratings will not display correctly.